README.rst

```diff
@@ -103,114 +103,69 @@ Alternatively, you can explicitly start and end a span:
 Customization
 -------------
 
-Samplers
-~~~~~~~~
+There are several things you can customize how OpenCensus works:
 
-You can specify different samplers when initializing a tracer, default
-is using ``AlwaysOnSampler``, the other options are ``AlwaysOffSampler``
-and ``ProbabilitySampler``
+* **Blacklist**, which excludes certain hosts and paths from being tracked.
+  By default, the health check path for the App Engine flexible environment is
+  not tracked, you can turn it on by excluding it from the blacklist setting.
 
-.. code:: python
-
-    from opencensus.trace.samplers import probability
-    from opencensus.trace import tracer as tracer_module
-
-    # Sampling the requests at the rate equals 0.5
-    sampler = probability.ProbabilitySampler(rate=0.5)
-    tracer = tracer_module.Tracer(sampler=sampler)
-
-Exporters
-~~~~~~~~~
-
-By default, the traces are printed to stdout in JSON format. You can choose
-different exporters to send the traces to. There are three built-in exporters,
-which are ``opencensus.trace.print_exporter``, ``opencensus.trace.file_exporter``
-and ``opencensus.trace.logging_exporter``, other exporters are provided as
-`extensions <#trace-exporter>`__.
-
-This example shows how to configure OpenCensus to save the traces to a
-file:
-
-.. code:: python
-
-    from opencensus.trace import file_exporter
-    from opencensus.trace.tracers import context_tracer
+* **Exporter**, which sends the telemetry data.
+  By default, the traces are printed to stdout in JSON format. You can choose
+  different exporters to send the traces to. There are three built-in exporters,
+  which are ``PrintExporter``, ``FileExporter`` and ``LoggingExporter``, the
+  other exporters are provided as `extensions <#trace-exporter>`__.
 
-    exporter = file_exporter.FileExporter(file_name='traces')
-    tracer = context_tracer.ContextTracer(exporter=exporter)
+* **Sampler**, which determines how telemetry data got sampled.
+  By default is using ``AlwaysOnSampler``, the other options are
```

Comment:
"The default sampler is the ``AlwaysOnSampler``, other samplers include the ``AlwaysOffSampler`` and ``ProbabilitySampler``."